### DISCUSSION TITLE
Freetype Bindings

### DISCUSSION THREAD

**Discussion by englerj** [odin-lang/Odin#1212] [General]
Is there a Freetype binding in the works somewhere? I noticed there was a [gist](https://gist.github.com/gingerBill/264e726a578a0e7e9128f38f3375b305), but it was from a few years ago. I was thinking about starting a new repo.

**Comment by Kelimion** [comment]
FreeType bindings exist, to a degree, as part of `vendor:sdl2/ttf` in that `SDL_ttf ` itself binds  FreeType and HarfBuzz. But there are no (official) direct bindings for it.

**Reply by englerj** [reply]
I went ahead and [started one](https://github.com/englerj/odin-freetype). Is it frowned upon to add incomplete bindings to the Odin Libs wiki section?
